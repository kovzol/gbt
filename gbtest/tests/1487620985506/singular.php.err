0.00user 0.00system 0:00.00elapsed 88%CPU (0avgtext+0avgdata 10072maxresident)k
0inputs+8outputs (0major+418minor)pagefaults 0swaps
