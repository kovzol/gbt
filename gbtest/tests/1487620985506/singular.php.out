                     SINGULAR                                 /
 A Computer Algebra System for Polynomial Computations       /   version 4.0.3
                                                           0<
 by: W. Decker, G.-M. Greuel, G. Pfister, H. Schoenemann     \   Jan 2016
FB Mathematik der Universitaet, D-67653 Kaiserslautern        \  Debian 4.0.3+ds-1
_[1]=25*v4-13
_[2]=25*v3-113
_[3]=25*v2-13
_[4]=50*v1+63
_[5]=62500*v5*v6^2-65000*v5*v6-101875*v6^2+16900*v5+105950*v6-27547
> Auf Wiedersehen.
